software/retrom.yml

```
@@ -0,0 +1,12 @@
+name: "Retrom"
+website_url: "https://github.com/JMBeresford/retrom"
+source_code_url: "https://github.com/JMBeresford/retrom"
+description: "Private cloud game library distribution server + frontend/launcher"
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Rust
+  - Javascript
```

This does not seem to be a server-side requirement.
```suggestion
  - Javascript
```